spec/system/provider_interface/export_applications_in_hesa_format_spec.rb

```diff
@@ -30,10 +30,13 @@ def and_my_organisation_has_courses_with_applications
     course = create(:course, provider: providers.first)
     course_option = create(:course_option, course:)
     @applications = create_list(:application_choice,
-                                5,
+                                10,
```

Comment:
Do we need to create 10 Applications? Could we get away with leaving this as 5, or worst case 6?

Reply:
Yes, I increased but it can be with 5. I will put it back :)